src/core/qgsdistancearea.h

```diff
@@ -222,6 +222,21 @@ class CORE_EXPORT QgsDistanceArea
      */
     double measureLine( const QgsPoint &p1, const QgsPoint &p2, QgsUnitTypes::DistanceUnit &units ) const;
 
+    /**
+     * calculates distance from one point with distance in meters and azimuth (direction)
+     * based on inverse Vincenty's formula when Geographic
+     * otherwise QgsPoint.project() will be called after QgsUnitTypes::fromUnitToUnitFactor() has been applied to the distance
+     * note: usage:
+     *  QgsDistanceArea myDa;
+      * myDa.setSourceCrs( layer->crs() );
+     * \since QGIS 3.0
+     * @param p1 start point [can be Cartesian or Geographic]
```

Comment:
Yes - all the @ tags need to be converted to \ tags. I.e. @note -> \note, @see -> \see, etc.

> Also, should '\since since QGIS 3.0' be place at a specific position (i.e. last entry?)

I don't think it makes any difference.